pom.xml

```diff
@@ -94,6 +94,16 @@
       <version>2.6.0</version>
     </dependency>
     <dependency>
+      <groupId>io.zipkin.brave</groupId>
+      <artifactId>brave-okhttp</artifactId>
+      <version>3.13.0</version>
+    </dependency>
+    <dependency>
+      <groupId>io.zipkin.brave</groupId>
+      <artifactId>brave-spancollector-http</artifactId>
```

Comment:
So in brave's latest snapshot, you can use io.zipkin.reporter:zipkin-sender-okhttp to send spans to zipkin if you like. This feature will be released this week.